### PR TITLE
[framework] Remove redundant path segments (part 1)

### PR DESCRIPTION
**Pull request by nemrism**
Remove redundant path in relative links inside /framework/** files

Contributes to #10653

### CODE REVIEW

**Comment by mairaw** [comment]
I know these changes are super straightforward but let's try to keep these PRs with ~100 files. I'll review this one as-is though.

File: docs/framework/additional-apis/index.md [hunk 1]
```diff
@@ -39,22 +39,22 @@ These APIs support the product infrastructure and are not intended/supported to
...
+| [System.Net.ServicePoint.m\_ConnectionGroupList Field](m_connectiongrouplist.md) |
+| [System.Net.ServicePointManager.s\_ServicePointTable Field](s_servicepointtable.md) |
+| [System.Windows.Diagnostics.VisualDiagnostics.s\_isDebuggerCheckDisabledForTestPurposes Field](s-isdebuggercheckdisabledfortestpurposes-field.md) |
+| [System.Windows.Forms.Design.DataMemberFieldEditor Class](datamemberfieldeditor-class.md) |
+| [System.Windows.Forms.Design.DataMemberListEditor Class](datamemberlisteditor-class.md) |
   
 ## See also
 
-- [The .NET Framework and Out-of-Band Releases](../../../docs/framework/get-started/the-net-framework-and-out-of-band-releases.md)
+- [The .NET Framework and Out-of-Band Releases](the-net-framework-and-out-of-band-releases.md)
```
Comment (mairaw) [inline docs/framework/additional-apis/index.md]: I think this path needs to look like this:

```suggestion
- [The .NET Framework and Out-of-Band Releases](../get-started/the-net-framework-and-out-of-band-releases.md)
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Wonderful to see these changes. In addition to the broken link I've pointed out, I also have a few additional comments. I've reviewed 200/502 files so far.

File: docs/framework/network-programming/using-an-asynchronous-server-socket.md [hunk 2]
```diff
@@ -264,7 +264,7 @@ public static void ReadCallback(IAsyncResult ar)
   
 ## See also
 
-- [Using a Synchronous Server Socket](../../../docs/framework/network-programming/using-a-synchronous-server-socket.md)
-- [Asynchronous Server Socket Example](../../../docs/framework/network-programming/asynchronous-server-socket-example.md)
+- [Using a Synchronous Server Socket](using-a-synchronous-server-socket.md)
+- [Asynchronous Server Socket Example](asynchronous-server-socket-example.md)
 - [Threading](../../../docs/standard/threading/index.md)
```
Comment (mairaw) [inline docs/framework/network-programming/using-an-asynchronous-server-socket.md]:
```suggestion
- [Threading](../../standard/threading/index.md)
```

File: docs/framework/performance/etw-events.md [hunk 3]
```diff
@@ -14,14 +14,14 @@ Event tracing for Windows (ETW) is a high-performance, low-overhead, scalable tr
  In the .NET Framework, ETW event tracing is available for the common language runtime (CLR), the [Task Parallel Library](../../../docs/standard/parallel-programming/task-parallel-library-tpl.md), and [Parallel LINQ (PLINQ)](../../../docs/standard/parallel-programming/parallel-linq-plinq.md).
```
Comment (mairaw) [inline docs/framework/performance/etw-events.md]:
```suggestion
 In the .NET Framework, ETW event tracing is available for the common language runtime (CLR), the [Task Parallel Library](../../standard/parallel-programming/task-parallel-library-tpl.md), and [Parallel LINQ (PLINQ)](../../standard/parallel-programming/parallel-linq-plinq.md).  
```

File: docs/framework/performance/etw-events.md [hunk 4]
```diff
@@ -14,14 +14,14 @@ Event tracing for Windows (ETW) is a high-performance, low-overhead, scalable tr
...
   
- [ETW Events in the Common Language Runtime](../../../docs/framework/performance/etw-events-in-the-common-language-runtime.md)  
+ [ETW Events in the Common Language Runtime](etw-events-in-the-common-language-runtime.md)  
  Describes how CLR ETW events supplement the profiling and debugging support provided by the common language runtime.  
   
 ## See also
 
-- [CLR ETW Events](../../../docs/framework/performance/clr-etw-events.md)
+- [CLR ETW Events](clr-etw-events.md)
 - [Task Parallel Library (TPL)](../../../docs/standard/parallel-programming/task-parallel-library-tpl.md)
```
Comment (mairaw) [inline docs/framework/performance/etw-events.md]:
```suggestion
- [Task Parallel Library (TPL)](../../standard/parallel-programming/task-parallel-library-tpl.md)
```

File: docs/framework/performance/lazy-initialization.md [hunk 5]
```diff
@@ -151,4 +151,4 @@ ms.author: "ronpet"
 - [Managed Threading Basics](../../../docs/standard/threading/managed-threading-basics.md)
```
Comment (mairaw) [inline docs/framework/performance/lazy-initialization.md]:
```suggestion
- [Managed Threading Basics](../../standard/threading/managed-threading-basics.md)
```

File: docs/framework/performance/lazy-initialization.md [hunk 6]
```diff
@@ -151,4 +151,4 @@ ms.author: "ronpet"
...
 - [Threads and Threading](../../../docs/standard/threading/threads-and-threading.md)
```
Comment (mairaw) [inline docs/framework/performance/lazy-initialization.md]:
```suggestion
- [Threads and Threading](../../standard/threading/threads-and-threading.md)
```

File: docs/framework/performance/lazy-initialization.md [hunk 7]
```diff
@@ -151,4 +151,4 @@ ms.author: "ronpet"
...
 - [Task Parallel Library (TPL)](../../../docs/standard/parallel-programming/task-parallel-library-tpl.md)
```
Comment (mairaw) [inline docs/framework/performance/lazy-initialization.md]:
```suggestion
- [Task Parallel Library (TPL)](../../standard/parallel-programming/task-parallel-library-tpl.md)
```

File: docs/framework/reflection-and-codedom/dynamic-language-runtime-overview.md [hunk 8]
```diff
@@ -70,7 +70,7 @@ Examples of languages developed by using the DLR include the following:
 ## DLR Architecture
  The following illustration shows the architecture of the dynamic language runtime.
 
- ![Dynamic Language Runtime Architecture Overview](../../../docs/framework/reflection-and-codedom/media/dlr-archoverview.png "DLR_ArchOverview")
+ ![Dynamic Language Runtime Architecture Overview](media/dlr-archoverview.png "DLR_ArchOverview")
```
Comment (mairaw) [inline docs/framework/reflection-and-codedom/dynamic-language-runtime-overview.md]:
```suggestion
 ![Dynamic Language Runtime Architecture Overview](./media/dlr-archoverview.png "DLR_ArchOverview")
```

File: docs/framework/reflection-and-codedom/dynamically-loading-and-using-types.md [hunk 9]
```diff
@@ -97,5 +97,5 @@ End Module
 
 - <xref:System.Type.InvokeMember%2A?displayProperty=nameWithType>
 - <xref:System.Reflection.Assembly.Load%2A?displayProperty=nameWithType>
-- [Viewing Type Information](../../../docs/framework/reflection-and-codedom/viewing-type-information.md)
+- [Viewing Type Information](viewing-type-information.md)
 - [Type Conversion in the .NET Framework](../../../docs/standard/base-types/type-conversion.md)
```
Comment (mairaw) [inline docs/framework/reflection-and-codedom/dynamically-loading-and-using-types.md]:
```suggestion
- [Type Conversion in the .NET Framework](../../standard/base-types/type-conversion.md)
```

**Review by mairaw — COMMENTED** [review]
A few more comments... 300/502 files reviewed.

File: docs/framework/tools/winres-exe-windows-forms-resource-editor.md [hunk 10]
```diff
@@ -138,6 +138,6 @@ You should use the version of Winres.exe that was released with the .NET Framewo
 - <xref:System.Resources.ResourceManager>
 - <xref:System.Resources.ResourceReader>
 - <xref:System.Resources.ResourceWriter>
-- [Tools](../../../docs/framework/tools/index.md)
-- [Resources in Desktop Apps](../../../docs/framework/resources/index.md)
+- [Tools](index.md)
+- [Resources in Desktop Apps](../resources/index.md)
 - [Globalization and Localization](../../../docs/standard/globalization-localization/index.md)
```
Comment (mairaw) [inline docs/framework/tools/winres-exe-windows-forms-resource-editor.md]:
```suggestion
- [Globalization and Localization](../../standard/globalization-localization/index.md)
```

File: docs/framework/ui-automation/access-embedded-objects-using-ui-automation.md [hunk 11]
```diff
@@ -21,7 +21,7 @@ ms.assetid: a5b513ec-7fa6-4460-869f-c18ff04f7cf2
   
  Embedded objects are considered children of the [!INCLUDE[TLA2#tla_uiautomation](../../../includes/tla2sharptla-uiautomation-md.md)] text provider. This allows them to be exposed through the same UI Automation tree structure as all other [!INCLUDE[TLA#tla_ui](../../../includes/tlasharptla-ui-md.md)] elements. Functionality, in turn, is exposed through the control patterns typically required by the embedded objects control type (for example, since hyperlinks are text-based they will support <xref:System.Windows.Automation.TextPattern>).  
   
- ![Embedded objects in a text container.](../../../docs/framework/ui-automation/media/uia-textpattern-embeddedobjects.PNG "UIA_TextPattern_EmbeddedObjects")  
+ ![Embedded objects in a text container.](media/uia-textpattern-embeddedobjects.PNG "UIA_TextPattern_EmbeddedObjects")
```
Comment (mairaw) [inline docs/framework/ui-automation/access-embedded-objects-using-ui-automation.md]:
```suggestion
 ![Embedded objects in a text container.](./media/uia-textpattern-embeddedobjects.PNG "UIA_TextPattern_EmbeddedObjects")  
```

**Review by mairaw — COMMENTED** [review]
More comments,,, 400/502

File: docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md [hunk 12]
```diff
@@ -28,12 +28,12 @@ ms.assetid: 880cd85c-aa8c-4fb5-9369-45491d34bb78
 > [!NOTE]
 >  This concept becomes evident in a [!INCLUDE[TLA#tla_xl](../../../includes/tlasharptla-xl-md.md)] spreadsheet where a user has defined a "First name" column. This column now has two headers—the "First name" header defined by the user and the alphanumeric designation for that column assigned by the application.  
   
-- See [Implementing the UI Automation Grid Control Pattern](../../../docs/framework/ui-automation/implementing-the-ui-automation-grid-control-pattern.md) for related grid functionality.  
+- See [Implementing the UI Automation Grid Control Pattern](implementing-the-ui-automation-grid-control-pattern.md) for related grid functionality.  
   
- ![Table with complex header items.](../../../docs/framework/ui-automation/media/uia-tablepattern-complex-column-headers.PNG "UIA_TablePattern_Complex_Column_Headers")  
+ ![Table with complex header items.](media/uia-tablepattern-complex-column-headers.PNG "UIA_TablePattern_Complex_Column_Headers")
```
Comment (mairaw) [inline docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md]:
```suggestion
 ![Table with complex header items.](./media/uia-tablepattern-complex-column-headers.PNG "UIA_TablePattern_Complex_Column_Headers")  
```

File: docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md [hunk 13]
```diff
@@ -28,12 +28,12 @@ ms.assetid: 880cd85c-aa8c-4fb5-9369-45491d34bb78
...
   
-- See [Implementing the UI Automation Grid Control Pattern](../../../docs/framework/ui-automation/implementing-the-ui-automation-grid-control-pattern.md) for related grid functionality.  
+- See [Implementing the UI Automation Grid Control Pattern](implementing-the-ui-automation-grid-control-pattern.md) for related grid functionality.  
   
- ![Table with complex header items.](../../../docs/framework/ui-automation/media/uia-tablepattern-complex-column-headers.PNG "UIA_TablePattern_Complex_Column_Headers")  
+ ![Table with complex header items.](media/uia-tablepattern-complex-column-headers.PNG "UIA_TablePattern_Complex_Column_Headers")  
 Example of a Table with Complex Column Headers  
   
- ![Table with ambiguous RowOrColumnMajor property.](../../../docs/framework/ui-automation/media/uia-tablepattern-roworcolumnmajorproperty.PNG "UIA_TablePattern_RowOrColumnMajorProperty")  
+ ![Table with ambiguous RowOrColumnMajor property.](media/uia-tablepattern-roworcolumnmajorproperty.PNG "UIA_TablePattern_RowOrColumnMajorProperty")
```
Comment (mairaw) [inline docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md]:
```suggestion
 ![Table with ambiguous RowOrColumnMajor property.](./media/uia-tablepattern-roworcolumnmajorproperty.PNG "UIA_TablePattern_RowOrColumnMajorProperty")  
```

File: docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md [hunk 14]
```diff
@@ -13,15 +13,15 @@ ms.assetid: b0fcdd87-3add-4345-bca9-e891205e02ba
...
   
 <a name="Implementation_Guidelines_and_Conventions"></a>   
 ## Implementation Guidelines and Conventions  
  When implementing the Value control pattern, note the following guidelines and conventions:  
   
 - Controls such as <xref:System.Windows.Automation.ControlType.ListItem> and <xref:System.Windows.Automation.ControlType.TreeItem> must support <xref:System.Windows.Automation.ValuePattern> if the value of any of the items is editable, regardless of the current edit mode of the control. The parent control must also support <xref:System.Windows.Automation.ValuePattern> if the child items are editable.  
   
- ![Editable list item.](../../../docs/framework/ui-automation/media/uia-valuepattern-editable-listitem.PNG "UIA_ValuePattern_Editable_ListItem")  
+ ![Editable list item.](media/uia-valuepattern-editable-listitem.PNG "UIA_ValuePattern_Editable_ListItem")
```
Comment (mairaw) [inline docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md]:
```suggestion
 ![Editable list item.](./media/uia-valuepattern-editable-listitem.PNG "UIA_ValuePattern_Editable_ListItem")  
```

File: docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md [hunk 15]
```diff
@@ -32,7 +32,7 @@ Example of an Editable List Item
   
 - <xref:System.Windows.Automation.Provider.IValueProvider> must be implemented by controls such as the **Color Picker** selection control from [!INCLUDE[TLA#tla_word](../../../includes/tlasharptla-word-md.md)] (illustrated below), which supports string mapping between a color value (for example, "yellow") and an equivalent internal [!INCLUDE[TLA#tla_rgb](../../../includes/tlasharptla-rgb-md.md)] structure.  
   
- ![Color picker with yellow highlighted.](../../../docs/framework/ui-automation/media/uia-valuepattern-colorpicker.png "UIA_ValuePattern_ColorPicker")  
+ ![Color picker with yellow highlighted.](media/uia-valuepattern-colorpicker.png "UIA_ValuePattern_ColorPicker")
```
Comment (mairaw) [inline docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md]:
```suggestion
 ![Color picker with yellow highlighted.](./media/uia-valuepattern-colorpicker.png "UIA_ValuePattern_ColorPicker")  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md [hunk 16]
```diff
@@ -20,10 +20,10 @@ ms.assetid: 93fdfbb9-0025-4b72-8ca0-0714adbb70d5
 ## Embedded Objects and the UI Automation Tree  
  Embedded objects are treated as individual elements within the control view of the [!INCLUDE[TLA2#tla_uiautomation](../../../includes/tla2sharptla-uiautomation-md.md)] tree. They are exposed as children of the text container so that they can be accessed through the same model as other controls in [!INCLUDE[TLA2#tla_uiautomation](../../../includes/tla2sharptla-uiautomation-md.md)].  
   
- ![Embedded Table with Image in a Text Container](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-example1.png "UIA_TextPattern_Embedded_Objects_Overview_Example1")  
+ ![Embedded Table with Image in a Text Container](media/uia-textpattern-embedded-objects-overview-example1.png "UIA_TextPattern_Embedded_Objects_Overview_Example1")
```
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 ![Embedded Table with Image in a Text Container](./media/uia-textpattern-embedded-objects-overview-example1.png "UIA_TextPattern_Embedded_Objects_Overview_Example1")  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md [hunk 17]
```diff
@@ -20,10 +20,10 @@ ms.assetid: 93fdfbb9-0025-4b72-8ca0-0714adbb70d5
...
   
- ![Embedded Table with Image in a Text Container](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-example1.png "UIA_TextPattern_Embedded_Objects_Overview_Example1")  
+ ![Embedded Table with Image in a Text Container](media/uia-textpattern-embedded-objects-overview-example1.png "UIA_TextPattern_Embedded_Objects_Overview_Example1")  
 Example of a Text Container with Table, Image, and Hyperlink Embedded Objects  
   
- ![Content view for the preceding example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-example2.PNG "UIA_TextPattern_Embedded_Objects_Overview_Example2")  
+ ![Content view for the preceding example](media/uia-textpattern-embedded-objects-overview-example2.PNG "UIA_TextPattern_Embedded_Objects_Overview_Example2")
```
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 ![Content view for the preceding example](./media/uia-textpattern-embedded-objects-overview-example2.PNG "UIA_TextPattern_Embedded_Objects_Overview_Example2")  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md [hunk 18]
```diff
@@ -32,7 +32,7 @@ Example of the Content View for a Portion of the Preceding Text Container
...
   
- ![Text ranges spanned by embedded objects.](../../../docs/framework/ui-automation/media/uia-textpattern-embeddedobjecttextranges.png "UIA_TextPattern_EmbeddedObjectTextRanges")  
+ ![Text ranges spanned by embedded objects.](media/uia-textpattern-embeddedobjecttextranges.png "UIA_TextPattern_EmbeddedObjectTextRanges")
```
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 ![Text ranges spanned by embedded objects.](./media/uia-textpattern-embeddedobjecttextranges.png "UIA_TextPattern_EmbeddedObjectTextRanges")  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md [hunk 19]
```diff
@@ -45,7 +45,7 @@ Example of a text stream with embedded objects and their range spans
   
 4. The range is then expanded from a degenerate range state by moving the <xref:System.Windows.Automation.Text.TextPatternRangeEndpoint.End> endpoint by one requested <xref:System.Windows.Automation.Text.TextUnit> boundary.  
   
- ![Range adjustments by Move & ExpandToEnclosingUnit](../../../docs/framework/ui-automation/media/uia-textpattern-moveandexpand-examples.png "UIA_TextPattern_MoveAndExpand_Examples")  
+ ![Range adjustments by Move & ExpandToEnclosingUnit](media/uia-textpattern-moveandexpand-examples.png "UIA_TextPattern_MoveAndExpand_Examples")
```
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 ![Range adjustments by Move & ExpandToEnclosingUnit](./media/uia-textpattern-moveandexpand-examples.png "UIA_TextPattern_MoveAndExpand_Examples")  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md [hunk 20]
```diff
@@ -95,18 +95,18 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
 ### Image  
  **Example 1 - A text range that contains an embedded image**  
   
- {The image ![Embedded Image Example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text}.  
+ {The image ![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text}.
```
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 {The image ![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text}.  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md [hunk 21]
```diff
@@ -95,18 +95,18 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
...
+|<xref:System.Windows.Automation.TextPattern.RangeFromChild%2A> where <xref:System.Windows.Automation.AutomationElement> is the object returned by the previous <xref:System.Windows.Automation.Text.TextPatternRange.GetChildren%2A> method.|Returns the degenerate range that represents "![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")".|  
   
  **Example 2 - A text range that partially spans the content of a text container. The text container has an embedded image that is not part of the text range.**  
   
- {The image} ![Embedded Image Example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text.  
+ {The image} ![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text.
```
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 {The image} ![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text.  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md [hunk 22]
```diff
@@ -121,16 +121,16 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
   
 |Cell with Image|Cell with Text|  
 |---------------------|--------------------|  
-|![Embedded Image Example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
-|![Embedded Image Example 2](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|  
-|![Embedded Image Example 3](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample3.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample3")<br /><br /> Image for Z|Z|  
+|![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|
```
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
|![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md [hunk 23]
```diff
@@ -121,16 +121,16 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
   
 |Cell with Image|Cell with Text|  
 |---------------------|--------------------|  
-|![Embedded Image Example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
-|![Embedded Image Example 2](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|  
-|![Embedded Image Example 3](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample3.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample3")<br /><br /> Image for Z|Z|  
+|![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
+|![Embedded Image Example 2](media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|
```
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
|![Embedded Image Example 2](./media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md [hunk 24]
```diff
@@ -121,16 +121,16 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
   
 |Cell with Image|Cell with Text|  
 |---------------------|--------------------|  
-|![Embedded Image Example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
-|![Embedded Image Example 2](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|  
-|![Embedded Image Example 3](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample3.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample3")<br /><br /> Image for Z|Z|  
+|![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
+|![Embedded Image Example 2](media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|  
+|![Embedded Image Example 3](media/uia-textpattern-embedded-objects-overview-imageexample3.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample3")<br /><br /> Image for Z|Z|
```
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
|![Embedded Image Example 3](./media/uia-textpattern-embedded-objects-overview-imageexample3.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample3")<br /><br /> Image for Z|Z|  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md [hunk 25]
```diff
@@ -121,16 +121,16 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
...
   
  **Example 1 - Get the text container from the content of a cell.**  
   
 |Method Called|Result|  
 |-------------------|------------|  
 |<xref:System.Windows.Automation.GridPattern.GetItem%2A> with parameters (0,0)|Returns the <xref:System.Windows.Automation.AutomationElement> representing the content of the table cell; in this case, the element is a text control.|  
-|<xref:System.Windows.Automation.TextPattern.RangeFromChild%2A> where <xref:System.Windows.Automation.AutomationElement> is the object returned by the previous `GetItem` method.|Returns the range that spans the image ![Embedded Image Example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample").|  
+|<xref:System.Windows.Automation.TextPattern.RangeFromChild%2A> where <xref:System.Windows.Automation.AutomationElement> is the object returned by the previous `GetItem` method.|Returns the range that spans the image ![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample").|
```
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
|<xref:System.Windows.Automation.TextPattern.RangeFromChild%2A> where <xref:System.Windows.Automation.AutomationElement> is the object returned by the previous `GetItem` method.|Returns the range that spans the image ![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample").|  
```

File: docs/framework/ui-automation/ui-automation-support-for-the-pane-control-type.md [hunk 26]
```diff
@@ -81,7 +81,7 @@ ms.assetid: 79761191-4449-4630-899c-9cbdb8867d3f
 ## Pane Control Type Example  
  The following image illustrates a control that implements the Pane control type.  
   
- ![Screenshot of applet window with two panes](../../../docs/framework/ui-automation/media/uiauto-pane.GIF "uiauto_pane")  
+ ![Screenshot of applet window with two panes](media/uiauto-pane.GIF "uiauto_pane")
```
Comment (mairaw) [inline docs/framework/ui-automation/ui-automation-support-for-the-pane-control-type.md]:
```suggestion
 ![Screenshot of applet window with two panes](./media/uiauto-pane.GIF "uiauto_pane")  
```

File: docs/framework/ui-automation/ui-automation-support-for-the-splitbutton-control-type.md [hunk 27]
```diff
@@ -70,7 +70,7 @@ ms.assetid: 14b05ccf-bcd8-4045-9bae-f7679cd98711
 ## SplitButton Control Example  
  The following image illustrates a SplitButton control type in a data grid control.  
   
- ![Split button](../../../docs/framework/ui-automation/media/uiauto-splitbutton-detailed.gif "uiauto_splitbutton_detailed")  
+ ![Split button](media/uiauto-splitbutton-detailed.gif "uiauto_splitbutton_detailed")
```
Comment (mairaw) [inline docs/framework/ui-automation/ui-automation-support-for-the-splitbutton-control-type.md]:
```suggestion
 ![Split button](./media/uiauto-splitbutton-detailed.gif "uiauto_splitbutton_detailed")  
```

File: docs/framework/ui-automation/ui-automation-textpattern-overview.md [hunk 28]
```diff
@@ -124,12 +124,12 @@ A formatting characteristic of a text range (for example, <xref:System.Windows.A
...
+There are two types of embedded objects in the [!INCLUDE[TLA2#tla_uiautomation](../../../includes/tla2sharptla-uiautomation-md.md)] text model. They consist of text-based content elements such as hyperlinks or tables, and control elements such as images and buttons. For more detailed information, see [Access Embedded Objects Using UI Automation](access-embedded-objects-using-ui-automation.md).
 
 **Endpoint**\
 The absolute <xref:System.Windows.Automation.Text.TextPatternRangeEndpoint.Start> or <xref:System.Windows.Automation.Text.TextPatternRangeEndpoint.End> point of a text range within a text container.
 
-![TextPatternRangeEndpoints &#40;start and end&#41;.](../../../docs/framework/ui-automation/media/uia-textpattern-endpoints.PNG "UIA_TextPattern_Endpoints")
+![TextPatternRangeEndpoints &#40;start and end&#41;.](media/uia-textpattern-endpoints.PNG "UIA_TextPattern_Endpoints")
```
Comment (mairaw) [inline docs/framework/ui-automation/ui-automation-textpattern-overview.md]:
```suggestion
![TextPatternRangeEndpoints &#40;start and end&#41;.](./media/uia-textpattern-endpoints.PNG "UIA_TextPattern_Endpoints")
```

File: docs/framework/windows-services/how-to-create-windows-services.md [hunk 29]
```diff
@@ -26,11 +26,11 @@ When you create a service, you can use a Visual Studio project template called *
 1. Create a **Windows Service** project.  
   
     > [!NOTE]
-    >  For instructions on writing a service without using the template, see [How to: Write Services Programmatically](../../../docs/framework/windows-services/how-to-write-services-programmatically.md).  
+    >  For instructions on writing a service without using the template, see [How to: Write Services Programmatically](how-to-write-services-programmatically.md).  
   
 2. In the **Properties** window, set the <xref:System.ServiceProcess.ServiceBase.ServiceName%2A> property for your service.  
   
-     ![Set the ServiceName property.](../../../docs/framework/windows-services/media/windowsservice-servicename.PNG "WindowsService_ServiceName")  
+     ![Set the ServiceName property.](media/windowsservice-servicename.PNG "WindowsService_ServiceName")
```
Comment (mairaw) [inline docs/framework/windows-services/how-to-create-windows-services.md]:
```suggestion
     ![Set the ServiceName property.](./media/windowsservice-servicename.PNG "WindowsService_ServiceName")  
```

**Review by mairaw — COMMENTED** [review]
Finished the review now. A couple more comments...

Also, apologies for including two commits here that were supposed to be done on a separate PR I'm working on. It's fine to leave them here if they don't cause any issues.

**Comment by nemrism** [comment]
I applied all suggested fixes about "incorrect" includes of `media` files.

I've changed all links to `/standard` section in the following PR: #13925 as this replacement work is quite huge even if I split by folders...
Once it will be merged I could rebase this branch in order to "facilitate" the final review.

**Comment by mairaw** [comment]
@nemrism can you fix the merge conflicts?

**Comment by nemrism** [comment]
Yes, I'll push that asap

**Comment by nemrism** [comment]
Merged, resolved and ready for your review @mairaw

**Comment by mairaw** [comment]
Thanks @nemrism. @rpetrusha I'm afraid this will cause new merge conflicts for #13851. Should we hold off until that one is merged?

**Comment by nemrism** [comment]
cc @mairaw / @rpetrusha 

Just tested the [v-thepet/move2](https://github.com/v-thepet/docs/tree/move2) branch against this one. It is indeed the most impacted one by merge conflicts but it is quite easy to resolve them.

Let me know when the #13851 PR will be merged, I'll handle the conflicts.

I'll continue to work on the #10653 resolution, it seems that other section are less impacted, automatic merge works fine on most of other related PR.

**Comment by mairaw** [comment]
That PR was now merged. Can you please rebase your PR? Thanks!

**Comment by nemrism** [comment]
Yes of course!
I'll prepare that and tell you when this one is ready for final review.

**Comment by nemrism** [comment]
All done, ready for final review @mairaw

**Review by mairaw — APPROVED** [review]
Changes look good! Thank you @nemrism!

**Pull request merged**